contracts/staking/StakingV1_1.sol

```diff
@@ -367,7 +289,7 @@ contract StakingV1_1 is StakingState, StakingConstants, PausableGuardian {
         internal
         returns (uint256 bzrxRewardsEarned, uint256 stableCoinRewardsEarned)
     {
-        bzrxRewardsEarned = _claimBzrx(restake);
+        bzrxRewardsEarned = _claimBzrx(false);
```

Comment:
restake argument is not using in function
function _claimBzrx(
        bool restake)
        internal
        returns (uint256 bzrxRewardsEarned)